packages/config/config/devices/0x0344/he-rs01.json

```diff
@@ -0,0 +1,862 @@
+					"value": 5
+				}
+			]
+		},
+		"63": {
+			"label": "Relay 3 External Input Number for Relay Output Control",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 3,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Controlled by gateway or associated device",
+					"value": 0
+				},
+				{
+					"label": "Controlled by external input S1",
+					"value": 1
+				},
+				{
+					"label": "Controlled by external input S2",
+					"value": 2
+				},
+				{
+					"label": "Controlled by external input S3",
+					"value": 3
+				},
+				{
+					"label": "Controlled by external input S4",
+					"value": 4
+				},
+				{
+					"label": "Controlled by external input S5",
+					"value": 5
+				}
+			]
+		},
+		"64": {
+			"label": "Relay 4 External Input Number for Relay Output Control",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 4,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Controlled by gateway or associated device",
+					"value": 0
+				},
+				{
+					"label": "Controlled by external input S1",
+					"value": 1
+				},
+				{
+					"label": "Controlled by external input S2",
+					"value": 2
+				},
+				{
+					"label": "Controlled by external input S3",
+					"value": 3
+				},
+				{
+					"label": "Controlled by external input S4",
+					"value": 4
+				},
+				{
+					"label": "Controlled by external input S5",
+					"value": 5
+				}
+			]
+		},
+		"65": {
+			"label": "Relay 5 External Input Number for Relay Output Control",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 5,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Controlled by gateway or associated device",
+					"value": 0
+				},
+				{
+					"label": "Controlled by external input S1",
+					"value": 1
+				},
+				{
+					"label": "Controlled by external input S2",
+					"value": 2
+				},
+				{
+					"label": "Controlled by external input S3",
+					"value": 3
+				},
+				{
+					"label": "Controlled by external input S4",
+					"value": 4
+				},
+				{
+					"label": "Controlled by external input S5",
+					"value": 5
+				}
+			]
+		},
+		"71": {
+			"label": "Relay 1 Timer Mode Duration",
+			"description": "This function can be used to open/close garage doors, blinds, curtains, etc. or to turn attached devices like door lock or security OFF for a short time. If the parameter value is set to ―0‖ it will operate as a short contact (about 0.5sec).",
+			"unit": "seconds",
+			"valueSize": 3,
+			"minValue": 0,
+			"maxValue": 43200,
+			"defaultValue": 0
+		},
+		"72": {
+			"label": "Relay 2 Timer Mode Duration",
+			"description": "This function can be used to open/close garage doors, blinds, curtains, etc. or to turn attached devices like door lock or security OFF for a short time. If the parameter value is set to ―0‖ it will operate as a short contact (about 0.5sec).",
```

Comment:
This description is too long. The "short" activation on value 0 should be an `option` with label `"0.5 seconds"` instead.

Reply:
Fixed